README.md

```diff
@@ -140,9 +143,23 @@ docker run --rm --volume $PWD:/md peterdavehello/markdownlint markdownlint --ign
 
 We use [PHPStan](https://phpstan.org/) for static code analysis.
 
-Running statis code analysis on a standalone Drupal module is a bit tricky, so we use a helper script to run the
+Running static code analysis on a standalone Drupal module is a bit tricky, so we use a helper script to run the
+analysis:
+
+```shell
+docker run --rm --volume ${PWD}:/app --workdir /app itkdev/php8.3-fpm ./scripts/code-analysis
+```
+
+## Unit tests
+
+We use [PHPUnit](https://phpunit.de/documentation.html) for unit testing.
+
+Testing mostly centers around the conversion and parsing of certificates. For this purpose a bunch of test
+certificates has been generated.
```

Comment:
Consider telling how the certificates have been created.

Reply:
Added section of certificate creation and extraction.